packages/openapi-generator/src/generator.ts

```diff
@@ -75,10 +74,23 @@ async function generateSDKSources(
   openApiDocument: OpenApiDocument,
   options: GeneratorOptions
 ): Promise<void> {
+  // TODO: This isn't really "request builder" anymore
   logger.info(`Generating request builder in ${serviceDir}.`);
   // TODO: what about overwrite?
+  if (openApiDocument.schemas.length) {
+    const modelDir = resolve(serviceDir, 'model');
+    await createInterfaceFiles(modelDir, openApiDocument);
+    await createFile(
```

Comment:
Perhaps you can create a follow up refactoring task, since I realized a few things which were not touched in this PR but are related and I would like to keep it nice from the start :-):
- Many methods have to many arguments e.g. `createFile()` One way would be that the `modelIndexFile` of anything else return not just a string but an interface containing also the `fileName` here 'index.ts' as well as the path where it is written.
- Also the `generateFromFile` has to many arguments. An the options come often together with the serviceName and the openpi content. Go over the code find common patterns and create perhaps a combined speaking object.

Reply:
- The input parameter of `generateFromFile` is just called `filePath` perhaps something like `inputFilePaht` is better.
- Also the `generateFromFile` does also a conversion and writing to a standard file I would draw this step out in a separate method.

So just create a follow up with `refactor method signatures and flow....`

Reply:
Makes sense

Reply:
Done.